### DISCUSSION TITLE
Let's see if we can develop a jailbreak for 11.00 based on TheFloW disclosed exploit

### DISCUSSION THREAD

**Discussion by ultimaweapon** [obhq/obliteration#857] [General]
Not sure if someone already started working on this. Here: https://twitter.com/theflow0/status/1783620012994945437

**Comment by ultimaweapon** [comment]
My test subject:
![PXL_20240427_123149643~2.jpg](https://github.com/obhq/obliteration/assets/3048307/3646568c-ee99-47d5-836a-a25a1c70e366)

**Comment by ultimaweapon** [comment]
The offset of `sppp_ipcp_RCR` function on 9.0 is `0x328A30`.

**Reply by ultimaweapon** [reply]
Found the location to cause heap buffer overwrite with `bcopy`. The first step is I'll try to crash the PS4 with this bug to see if it work on 10.50.

**Reply by ultimaweapon** [reply]
No, what I need to do first should be build a PPPoE server that is able to craft this malicious header.

**Comment by SoffiMeower** [comment]
@ultimaweapon If you get it working, then I guess I'll upgrade to 11.00 :P

**Comment by SoffiMeower** [comment]
Pspspspsps. @ultimaweapon 

https://github.com/TheOfficialFloW/PPPwn

**Reply by ultimaweapon** [reply]
Thanks!

**Reply by ultimaweapon** [reply]
Seems like this is very reliable. I just managed to success in a single run without any crashes.

**Reply by SoffiMeower** [reply]
To be fair, FloW is the one who discovered the exploit, told Sony so they could patch for later FW for legal reasons, then announced the exploit publicly just a week ago xD
